x-pack/plugins/canvas/public/components/app/app.js

```diff
@@ -6,11 +6,12 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
 import { routes } from '../../apps';
 import { shortcutManager } from '../../lib/shortcut_manager';
 import { Router } from '../router';
 
-export class App extends React.PureComponent {
+class AppUI extends React.PureComponent {
```

Comment:
We need to change component name to .`..Ui` when we use injectI18n HOC. We do not need to rename file because we still export `App` component at the bottom of the file